lib/beaker/host/windows.rb

```diff
@@ -50,12 +50,29 @@ def self.foss_defaults
         #let's just add both potential bin dirs to the path, include ruby too for `gem`, `ruby`, etc
         'puppetbindir'      => '/cygdrive/c/Program Files (x86)/Puppet Labs/Puppet/bin:/cygdrive/c/Program Files/Puppet Labs/Puppet/bin:/cygdrive/c/Program Files (x86)/Puppet Labs/Puppet/sys/ruby/bin:/cygdrive/c/Program Files/Puppet Labs/Puppet/sys/ruby/bin',
         'hierabindir'       => '/opt/puppet-git-repos/hiera/bin',
-        'pathseparator'     => ':',
+        'pathseparator'     => ';',
       })
     end
 
     def self.aio_defaults
-      self.foss_defaults
+      h = Beaker::Options::OptionsHash.new
+      h.merge({
+        'user'              => 'Administrator',
```

Comment:
`user` and `group` changes seem unrelated to QENG-1841 - pathing changes.  Not sure I understand how these are related?